### PR TITLE
CONTRIBUTING.md: remove excess verbiage

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,20 +5,20 @@ First off, thanks for your interest in contributing!
 ## Reporting bugs
 
 We exclusively use [bugzilla](https://issues.dlang.org/) for issue tracking, which is why Github issues are disabled on this repository.
-If you found a bug, please [check bugzilla](https://issues.dlang.org/query.cgi) to see if it's already reported.
-If it isn't, you can [create a new issue](https://issues.dlang.org/enter_bug.cgi).
+For reporting bugs, please [check bugzilla](https://issues.dlang.org/query.cgi) to see if it's already reported.
```
Comment (Geod24) [inline CONTRIBUTING.md]:
```suggestion
Before reporting bugs, please [check bugzilla](https://issues.dlang.org/query.cgi) to see if it's already reported.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -42,9 +42,9 @@ Note that after a version has entered the release window (there is a beta / the
 
 ## Solving bugs / Submitting pull requests
 
-Before submitting a PR there are some things you can check which will hopefully make the pulling process run smoothly.
+Before submitting a PR, checks to make the pulling process run smoothly are:
```
Comment (Geod24) [inline CONTRIBUTING.md:45]:
```suggestion
Before submitting a PR, check the following to make the pulling process run smoothly are:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -5,20 +5,20 @@ First off, thanks for your interest in contributing!
...
 
-If you have a question about a specific behavior, the [D.Learn](http://forum.dlang.org/group/learn) group is a good place to ask for clarification before reporting an issue.
+For questions about a specific behavior, the [D.Learn](http://forum.dlang.org/group/learn) group is a good place to ask for clarification before reporting an issue.
 
 ### Content
 
-When creating a new issue, make sure to include:
-- which version of DMD you are using (which can be found by running `dmd` with no argument).
+When creating a new issue, include:
+- the version of DMD being used (which can be found by running `dmd` with no argument).
```
Comment (thewilsonator) [inline CONTRIBUTING.md:16]: recommend `--version`?

**Review by Geod24 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (WalterBright) [reply CONTRIBUTING.md:16]: Simpler is better, and the version gets printed out if you just type dmd.

Comment (thewilsonator) [reply CONTRIBUTING.md:16]: fair enough.

File: CONTRIBUTING.md (hunk 2 above)
Comment (ibuclaw) [reply CONTRIBUTING.md:45]: "Follow these steps" also works as an active language instruction.

File: CONTRIBUTING.md (hunk 1 above)
Comment (WalterBright) [reply CONTRIBUTING.md]: Already fixed.

**Pull request merged**